exporter/cassandraexporter/exporter_logs.go

```diff
@@ -89,7 +89,10 @@ func (e *logsExporter) pushLogsData(ctx context.Context, ld plog.Logs) error {
 			for k := 0; k < rs.Len(); k++ {
 				r := rs.At(k)
 				logAttr := attributesToMap(r.Attributes().AsRaw())
-				bodyByte, _ := json.Marshal(r.Body().AsRaw())
+				bodyByte, err := json.Marshal(r.Body().AsRaw())
+				if err != nil {
+					return err
+				}
```

Comment:
This technically changes the behavior of the exporter. Before this change if the marshal caused an error then the exporter includes an empty string in the query params via `string(bodyByte)`.

Reply:
yes this change is desired. I think we should not let go of this error.